### DISCUSSION TITLE
Uncover in Nuclei, multiple engines

### DISCUSSION THREAD

**Discussion by HachimanSec** [projectdiscovery/nuclei#4180] [Q&A]
Hi there, 

I try to run uncover with multiple engines - according to https://github.com/projectdiscovery/nuclei

when I run:
`nuclei -fr -uc -uq SEARCHSTRING -ue shodan,censys,fofa,shodan-idb,quake,zoomeye,publicwww
`
it always faisl with:
```
[INF] Running uncover query against: shodan,censys,fofa,shodan-idb,quake,zoomeye,publicwww
[FTL] Could not create runner: could not create input provider: [uncover:RUNTIME] no agent/source specified
```

What is wrong with my syntax here? Anyone any idea?

Thx!

**Comment by MetzinAround** [comment]
Hi @HachimanSec , 

What happens if you run it with only one provider, like only censys? Does the error change? I'm thinking it might be something with the multiple providers.

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
